# Helper message for passing data about pixels in an image
int32 x
int32 y
int32 data    # optional data about this pixel
